spock-core/src/main/java/org/spockframework/runtime/ParameterizedFeatureNode.java

```diff
@@ -136,8 +143,67 @@ public Type getType() {
     @Override
     public void execute(TestDescriptor testDescriptor) {
       executionCounter.incrementAndGet();
-      addChild(testDescriptor);
-      delegate.execute(testDescriptor);
+      if (featureInfo.isReportIterations()) {
+        addChild(testDescriptor);
+        delegate.execute(testDescriptor);
+      } else {
+        //TODO: replace by result of https://github.com/junit-team/junit5/issues/2188
+        try {
```

Comment:
Could you move it to a separate method with a name clearly suggesting it is a workaround?

Reply:
Actually I don't expect this work-around to be merged to `master`, but once junit-team/junit5#2188 is done, that feature used instead. That's also why this PR is in draft state.
If we want to merge tis before junit-team/junit5#2188 is done, I can move it of course.

Reply:
@leonard84 what do you think?

Reply:
Lets wait a bit and see what the timeline for that JUnit feature is

Reply:
Ok, I moved the part that will be replaced once 2188 is implemented to a separate method, just in case. :-)

Reply:
And now it is replaced by the shiny new method in the milestone release :-)